packages/apollo-voyager-conflicts/src/handleConflict.ts

```diff
@@ -0,0 +1,49 @@
+import { SyncServerError } from './detectConflict'
+
+function RETURN_TO_CLIENT (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): SyncServerError {
```

Comment:
This is just plain javascript with added types. Let's drop function and also use valid typescript method names (non capitals and _)

Reply:
😆